pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1510,16 +1518,16 @@ func (c *Cloud) CreateDisk(volumeOptions *VolumeOptions) (string, error) {
 
 	case VolumeTypeIO1:
 		// See http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVolume.html for IOPS constraints
-		if volumeOptions.IOPSPerGB <= 0 || volumeOptions.IOPSPerGB > 30 {
-			return "", fmt.Errorf("invalid iopsPerGB value %d, must be 0 < IOPSPerGB <= 30", volumeOptions.IOPSPerGB)
+		if volumeOptions.IOPSPerGB <= 0 || volumeOptions.IOPSPerGB > MaxIOPSPerGB {
```

Comment:
IMO, we still need to have at least capping at 20000 IOPS, see below. I could live without `MaxIOPSPerGB`, however I don't think it hurts.